server/service/testing_client.go

```diff
@@ -45,6 +46,7 @@ type withServer struct {
 func (ts *withServer) SetupSuite(dbName string) {
 	ts.withDS.SetupSuite(dbName)
 
+	loginRateLimit = throttled.PerMin(100)
```

Comment:
The integration tests were starting to hit the standard limit, this increases it for tests.